.github/workflows/post-merge-ci.yml

```
@@ -363,9 +402,21 @@ jobs:
       operator_tag: ${{ needs.deploy-operator.outputs.operator_tag }}
     secrets: inherit
 
+  deploy-test-fastvideo:
+    needs: [deploy-operator, fastvideo-pipeline]
+    uses: ./.github/workflows/shared-deploy-test-framework.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/shared-deploy-test-framework.yml": open /home/jailuser/git/.github/workflows/shared-deploy-test-framework.yml: no such file or directory

(expression)